lib/open_api_spex/cast.ex

```diff
@@ -1,4 +1,6 @@
 defmodule OpenApiSpex.Cast do
+  @moduledoc "Validates values agains OpenApiSpex schemas"
```

Comment:
[nit] This isn't accurate. It "is responsible for casting and validating a value against the given schema."

Reply:
Updated it to reflect your comment